app-backend/api/src/main/scala/user/PseudoUsernameService.scala

```diff
@@ -0,0 +1,45 @@
+package com.rasterfoundry.api.user
+
+import com.rasterfoundry.datamodel.PseudoUsernameType
+
+import com.github.javafaker._
+
+import scala.util.Random
+
+import java.util.UUID
+
+object PseudoUsernameService {
+
+  def createPseudoName(peudoUserNameType: PseudoUsernameType): String = {
+    val uuidSeg =
+      UUID.randomUUID
+        .toString()
+        .split("-")
+        .toIndexedSeq(1 + (new Random).nextInt(3))
+    val faker = new Faker();
+    (peudoUserNameType match {
+      case PseudoUsernameType.GameOfThrones =>
+        s"${faker.gameOfThrones().character()} at ${faker.gameOfThrones().house()} ${uuidSeg}"
+      case PseudoUsernameType.HarryPotter =>
```

Comment:
Probably no one wants to be Voldemort or the Night King :see_no_evil: how much control does java faker give us over rejecting certain values? Probably doesn't matter unless someone tells us it does, but I'm curious

Reply:
Per discussions offline -- it's probably fine, we will add rejection of some certain names when requirements arise.